modules/ROOT/pages/endpoint-sftp-receive-target.adoc

```diff
@@ -1,91 +1,72 @@
 = SFTP Receive Endpoint Settings
 
-Use these settings to set up SFTP endpoints in one of the following contexts:
+You can use these settings to set up SFTP endpoints in one of the following contexts:
 
 * Receive from partners +
 Receive inbound EDI files from your trading partners used in inbound message flows.
 * Source at host +
 Receive application messages (in JSON or XML format) from your backend applications used in outbound message flows.
 
+You can create the STFP identifiers for your organization and for your trading partners in the respective xref:configure-host.adoc[host] and xref:configure-partner.adoc[trading partner] profile pages.
 
 [%header%autowidth.spread]
 |===
 |Name |Description | Required
-|Endpoint Name | The name is auto-populated, but you can overwrite it based on your organization’s naming conventions for consistency. 
+| Endpoint Name
+| The name used to identify the endpoint. The value in the `Endpoint Name` field is generated automatically, but can be modified manually.
```

Comment:
ditto prev. questions about UI labels vs. syntax parameter names or values

Reply:
DITTO previous edits to the table texts that follow:

Reply:
Done!